Funcky.Xunit/Extensions/ToTheoryDataExtension.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Funcky.Xunit
+{
+    public static class ToTheoryDataExtension
+    {
+        public static TheoryData<T1> ToTheoryData<T1>(this IEnumerable<T1> enumerable)
+            => enumerable.Aggregate(new TheoryData<T1>(), AddElementToTheoryData);
+
+        public static TheoryData<T1, T2> ToTheoryData<T1, T2>(this IEnumerable<Tuple<T1, T2>> enumerable)
```

Comment:
Why are you using `Tuple` instead of `ValueTuple`?

Reply:
Mafi had the same suggestion ;) I thouht I start with the typical Types, overloads with ValueTuple should be simple to add.

Reply:
But why support `Tuple`s at all, aren't most APIs based around `ValueTuple`s anyways?

Reply:
Examples off the top of my head:
* [Tuple syntax](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples)
* [Enumerable.Zip](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.zip?view=netcore-3.1#System_Linq_Enumerable_Zip__2_System_Collections_Generic_IEnumerable___0__System_Collections_Generic_IEnumerable___1__)

Reply:
Added ValueTuple overloads: 5bb6d2cd